static/templates/actors/character/tabs/actions.hbs

```diff
@@ -36,9 +36,9 @@
 
                     <ol class="actions-list item-list directory-list strikes-list" data-strikes>
                         {{#each data.actions as |strike ai|}}
-                            {{#unless (and strike.item.isStowed @root.actor.flags.pf2e.hideStowed)}}
+                            {{#if strike.visible}}
```

Comment:
This may break something that depends on index matching (though I've tried to eliminate that before)

Reply:
I'll trust you on that. Easy enough to go back on this.